src/process.js

```diff
@@ -164,13 +169,13 @@ class BaseProcess {
 
 	/* istanbul ignore next */
 	async execute(/*node*/) {
-		throw "execute not implemented yet";
+		throw new Error(`execute not implemented yet for process '${this.id}' (namespace: ${this.namespace})`);
```

Comment:
What if `this.namespace` is null?

Reply:
Although this should never occur to actual users and only to devs (who forget to implement the function), I've made it fall back to 'n/a', too.